src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -39,7 +39,6 @@
    :class:`AppStoreConnectCreateCredentialsEndpoint`.
 
 6. ``POST projects/{org_slug}/{proj_slug}/appstoreconnect/{id}/``
```

Comment:
it seems that this endpoint is no longer in the `sentry/src/sentry/api/urls.py ` file. I am getting 404.